### PR TITLE
Ref task #2779_2 Signup part 2

### PR DESCRIPTION
**Pull request by quang5566**
Signup

### CODE REVIEW

**Comment by quang5566** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -45,10 +48,19 @@ p {
...
-  margin-bottom: 15px;
   @include box_sizing;
 }
 
 input {
   height: auto !important;
 }
+
+/* footer*/
+#footer {
```
Comment (thanhhungchu95) [inline app/assets/stylesheets/custom.scss]: dùng class

**Comment by thanhhungchu95** [comment]
commented

**Comment by quang5566** [comment]
ready

**Pull request merged**